### ISSUE TITLE
Host own F-droid repo

### ISSUE THREAD

**Issue by rakshazi** [vector-im/element-android#2747]
Hello,
That feature request related to #2698 #2567

Please, host your own f-droid repo, like NewPipe did ([announcement](https://newpipe.net/blog/announcement/f-droid/pinned/f-droid-repo/), [instruction for users](https://newpipe.net/FAQ/tutorials/install-add-fdroid-repo/)), because installing APKs directy from CI/releases is not the best idea.

[How to setup F-droid repo](https://f-droid.org/en/docs/Setup_an_F-Droid_App_Repo/), [Repomaker - GUI tool for f-droid repos](https://f-droid.org/en/repomaker/)

**Comment by karolyi** [comment]
did something happen here? F-Droid still has 1.1.0 while 1.1.3 is already out for over a month now.

**Comment by Glitchy-Tozier** [comment]
I was wondering exactly the same thing

**Comment by rakshazi** [comment]
Regarding current outdated version (f-droid has only 1.1.0): https://github.com/vector-im/element-android/issues/3116

**Comment by HarHarLinks** [comment]
Please do this!

I want to add the argument that by doing so, you can also distribute your RCs (timely...) to your technical userbase (who know how to find and add a custom repo) and actually test them before release.

**Comment by PLAYERUNKNOWNHatsune01** [comment]
Please do it!

So many new features was released on element including (Spaces) And even on element blog post said. Download the latest app on f-droid but the element on f-froid still stuck on 1.1.6. Please make a repo

**Comment by karolyi** [comment]
again, at this point 1.1.10 is out 10 days ago, 1.1.11 is out 6 days ago, yet we're stuck on 1.1.9 (which is buggy) on F-Droid.

**Comment by rakshazi** [comment]
I suppose New Vector just doesn't have enough people to implement all suggestions and unfortunate you can't send a PR for this one :(

**Comment by karolyi** [comment]
NewPipe has a way of updating itself by checking its github repo and notifying you if there is a newer version, then it can download and itstall it via the browser. I use it that way.

That would be even less work than setting up an F-Droid repo.

**Comment by rakshazi** [comment]
NewPipe hosts their own F-Droid repo and provides notifications

**Comment by karolyi** [comment]
@rakshazi that doesn't make my comment invalid, or I have no idea as to why you've added your comment.

Kindly head over to https://github.com/TeamNewPipe/NewPipe/#installation-and-updates and see point 2.

I've seen newpipe updating itself via github on my phone, by notifying me from within itself that a new update is available.

**Comment by rakshazi** [comment]
No-no, your comment is valid,  I mean that NewPipe provides both features (f-droid repo **AND** release notifications) and that's very cool way

**Comment by Thatoo** [comment]
If I could, I'd test voice messages and spaces improvment but f-droid version is old... An f-droid repo would be indeed welcomed.
